pom.xml

```diff
@@ -30,21 +30,23 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.19</version>
+    <version>4.42</version>
   </parent>
 
   <artifactId>cloudbees-disk-usage-simple</artifactId>
-  <version>0.11-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
 
   <name>CloudBees Disk Usage Simple Plugin</name>
   <description>This is a simple disk usage plugin that calculates job disk usage while limiting the performance impact on the Jenkins master.</description>
   <url>https://github.com/jenkinsci/cloudbees-disk-usage-simple-plugin</url>
 
   <properties>
-    <jenkins.version>2.176.4</jenkins.version>
+    <revision>0.11</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <gitHubRepo>jenkinsci/cloudbees-disk-usage-simple-plugin</gitHubRepo>
+    <jenkins.version>2.319.3</jenkins.version>
```

Comment:
2.332.x would get rid of two additional implied dependencies on detached plugins: https://github.com/jenkinsci/jenkins/blob/de5b030a629d1a489b9a010a224a53453b40f237/core/src/main/resources/jenkins/split-plugins.txt#L38-L39

Reply:
2.332.4 it is :)